lib/src/controls/navigation/tab_view.dart

```diff
@@ -35,12 +42,20 @@ class TabView extends StatelessWidget {
     this.shortcutsEnabled = true,
     this.onReorder,
     this.showScrollButtons = true,
+    this.wheelScroll = false,
+    ScrollPosController? scrollPosController,
     this.minTabWidth = _kMinTileWidth,
     this.maxTabWidth = _kMaxTileWidth,
   })  : assert(tabs.length == bodies.length),
         assert(minTabWidth > 0 && maxTabWidth > 0),
         assert(minTabWidth < maxTabWidth),
-        super(key: key);
+        super(key: key) {
+    this.scrollPosController = scrollPosController ??
+        ScrollPosController(
+          itemCount: tabs.length,
+          animationDuration: const Duration(milliseconds: 100),
+        );
+  }
```

Comment:
It'd be better to handle this in a `StatefulWidget`. Also, `scrollPosController` should be disposed